typescript/infra/config/environments/mainnet3/tokenPrices.json

```diff
@@ -3,9 +3,11 @@
   "arbitrum": "2914.61",
   "avalanche": "35.71",
   "bsc": "373.19",
+  "blast": "2914.61",
```

Comment:
i'm seeing $14.56 on this site: https://coinranking.com/coin/xUvZdwsex+blast-blast

Reply:
think this is just the gas token prices, iirc blast uses eth for gas (ditto mode below)